### PR TITLE
VSTS-321 Adjust azure pipeline build with clean code taxonomy

### PR DESCRIPTION
**Pull request by stanislavhh**

### CODE REVIEW

File: commonv5/ts/sonarqube/Analysis.ts [hunk 1]
```diff
@@ -86,40 +107,33 @@ export default class Analysis {
...
       return "";
     }
-    const linkText = `Detailed ${this.endpointType} report &gt;`;
-    return `[${linkText}](${this.dashboardUrl})`;
+    const linkText = `Detailed ${this.endpointType} report`;
+
+    return `<div style="margin-top: 16px;">
+        <a href="${this.dashboardUrl}" style="${textStyle("#0067B8")}">
+        ${linkText}
+        <span class="bowtie-icon bowtie-navigate-external" style="color: #0067B8; font-size: 20px; vertical-align: middle;"></span>
```
Comment (7PH) [inline commonv5/ts/sonarqube/Analysis.ts:134]: I am a bit surprised that we do not use the lighter color for the link, but I did see that on figma it is like this, so I guess that's expected

File: commonv5/ts/sonarqube/Analysis.ts [hunk 2]
```diff
@@ -133,18 +147,13 @@ export default class Analysis {
...
-        return "#ed7d20";
-      case "ERROR":
-        return "#d4333f";
-      case "NONE":
-        return "#b4b4b4";
-      default:
-        return "#b4b4b4";
-    }
+    const colors = {
+      OK: "#6EB712",
```
Comment (7PH) [inline commonv5/ts/sonarqube/Analysis.ts:151]: How did you know what color to use there? I do not see it specified on figma?

**Review by 7PH — APPROVED** [review]
GG 👍

File: commonv5/ts/sonarqube/Analysis.ts [hunk 3]
```diff
@@ -133,18 +147,13 @@ export default class Analysis {
...
+      WARN: "#F5B840",
+      ERROR: "#D92D20",
```
[the remark this answers is not in the document]
Comment (stanislavhh) [reply commonv5/ts/sonarqube/Analysis.ts:153]: Yes, basically I was using colors in  [DS](https://www.figma.com/file/kuN046eNTwGnRwIsiP6Cjf/ProductDesignSystem?node-id=14678%3A5686&mode=dev) : bg-success, bg-warning, bg-error

File: commonv5/ts/sonarqube/Analysis.ts (hunk 2 above)
Comment (stanislavhh) [reply commonv5/ts/sonarqube/Analysis.ts:151]: I looked at the colors in [DS](https://www.figma.com/file/kuN046eNTwGnRwIsiP6Cjf/ProductDesignSystem?node-id=14678%3A5686&mode=dev) and background colors made sense to me and I double checked them with Simo as well

File: commonv5/ts/sonarqube/Analysis.ts (hunk 1 above)
Comment (stanislavhh) [reply commonv5/ts/sonarqube/Analysis.ts]: Good point, I double checked with Simo and we agreed on using color from DS, same for text color

**Review by 7PH — APPROVED** [review]

**Pull request merged**